packages/builder/src/steps/invoke.ts

```diff
@@ -245,10 +193,12 @@ export default {
   },
 
   configInject(ctx: ChainBuilderContextWithHelpers, config: Config) {
+    validateConfig('invoke', config);
+
     config = _.cloneDeep(config);
 
     if (config.target) {
-      config.target = config.target.map((v) => _.template(v)(ctx));
+      config.target = config.target.map((v) => _.template(v)(ctx)) as [string, ...string[]];
```

Comment:
comment explaining the use of unusual type would be helpful here. is this supposed to be a `string[]` with min length of 1?

Reply:
Will do. It means exactly what you described. This is the type that zod returns when setting that validation.